lib/process-request.js

```diff
@@ -1,6 +1,6 @@
 const url = require('url');
-const removeProperties = require('lodash.omit');
-const removeOtherProperties = require('lodash.pick');
+const removeProperties = require('lodash/omit');
+const removeOtherProperties = require('lodash/pick');
```

Comment:
Can you update the same `lodash.omit` and `lodash.pick` references in `lib/process-response.js` and add some tests for that too?

Reply:
hey @erunion, just replaced the individual `lodash` packages and added the tests for the nested properties.